Source/WebKit/WebProcess/WebPage/FindController.h

```diff
@@ -51,6 +51,8 @@ class CallbackID;
 class PluginView;
 class WebPage;
 
+enum class TriggerImageAnalysis : bool { No, Yes };
```

Comment:
Nit - maybe we should move this enum class into the `FindController` class scope? (Since it's a somewhat general name, but it has a very specific purpose here).